src/components/header/Header.js

```diff
@@ -1,19 +1,33 @@
 import styled from "styled-components";
 import { IoMdAdd } from "react-icons/io";
-import { BiFilterAlt } from "react-icons/bi";
 import { MdOutlineMoreVert } from "react-icons/md";
+import { BiSearch } from "react-icons/bi";
 
 const Header = (props) => {
 	return (
 		<HeaderContainer>
-			<HeaderTitle>{props.pageTitle}</HeaderTitle>
+			<HeaderTitle id="habitTitle">{props.pageTitle}</HeaderTitle>
 			<HeaderButtonContainer>
 				<AddButton onClick={props.handleOpen} title="add">
 					<IoMdAdd size={26} />
 				</AddButton>
-				<FilterButton>
-					<BiFilterAlt size={25} />
-				</FilterButton>
+				<SearchButton title="search">
+					<Container>
+						{props.showSearchField && (
```

Comment:
Lets create a separate component for search field.

Reply:
made a seperate component